lms/static/sass/shared-v2/_header.scss

```diff
@@ -125,3 +125,27 @@
         }
     }
 }
+
+.doc-link {
+    position: relative;
```

Comment:
I can remove more of these. I'm finding that font-size and font-weight are needed to match the style of the username link text that's next to it. Would it be OK to leave float right, margin, font-size, and font-weight? I agree that per-use styling isn't a great practice.